src/Diagnostics.RuntimeHost/Middleware/DiagnosticsRequestMiddleware.cs

```diff
@@ -87,12 +83,11 @@ private void BeginRequest_Handle(HttpContext httpContext)
             var supportBayApiObserverTokenService = ((ServiceProvider)httpContext.RequestServices).GetService<ISupportBayApiObserverTokenService>();
 
             httpContext.Items.Add(HostConstants.ApiLoggerKey, logger);
-            string clientObjId = string.Empty;
-            StringValues internalClientHeader;
-            bool isInternalClient = false;
+            var clientObjId = string.Empty;
+            var isInternalClient = false;
 
             // For requests coming Applens, populate client object id with Applens App Id.
-            httpContext.Request.Headers.TryGetValue("x-ms-internal-client", out internalClientHeader);
+            httpContext.Request.Headers.TryGetValue("x-ms-internal-client", out var internalClientHeader);
             if (internalClientHeader.Any())
```

Comment:
How would this work if x-ms-internal-client header is not found?

Reply:
This wouldn't change the functionality at all because the variable is still declared, just in a more compact manner; that said, it could be an issue if this out var came out null, so I'm updating the if statement to take the boolean result of TryGetValue into account as well.

Reply:
Ok. I will wait for the change.